File: VB/ConsoleApplication1/QuoteItem.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo

Namespace ConsoleApplication1

	Public Class QuoteItem
		Inherits XPObject
		Public Sub New()
			MyBase.New()
			' This constructor is used when an object is loaded from a persistent storage.
			' Do not place any code here.
		End Sub
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
			' This constructor is used when an object is loaded from a persistent storage.
			' Do not place any code here.
		End Sub
		Public Overrides Sub AfterConstruction()
			MyBase.AfterConstruction()
			' Place here your initialization code.
		End Sub
		Private _sequence As Integer
		Public Property Sequence() As Integer
			Get
				Return _sequence
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Sequence", _sequence, value)
			End Set
		End Property
		Private _quote As Quote
		<Association("Quote-QuoteItems")> _
		Public Property Quote() As Quote
			Get
				Return _quote
			End Get
			Set(ByVal value As Quote)
				SetPropertyValue("Quote", _quote, value)
			End Set
		End Property
	End Class
End Namespace

File: CS/ConsoleApplication1/QuoteItem.cs
using DevExpress.Xpo;

namespace ConsoleApplication1 {

    public class QuoteItem : XPObject {
        public QuoteItem()
            : base() {
            // This constructor is used when an object is loaded from a persistent storage.
            // Do not place any code here.
        }
        public QuoteItem(Session session)
            : base(session) {
            // This constructor is used when an object is loaded from a persistent storage.
            // Do not place any code here.
        }
        public override void AfterConstruction() {
            base.AfterConstruction();
            // Place here your initialization code.
        }
        private int _sequence;
        public int Sequence {
            get { return _sequence; }
            set { SetPropertyValue("Sequence", ref _sequence, value); }
        }
        private Quote _quote;
        [Association("Quote-QuoteItems")]
        public Quote Quote {
            get { return _quote; }
            set { SetPropertyValue("Quote", ref _quote, value); }
        }
    }
}